lib/rom/sql/relation.rb

```diff
@@ -39,6 +39,26 @@ def project(*names)
       end
 
       # @api public
+      def group(*args, &block)
+        __new__(dataset.__send__(__method__, *args, &block))
+      end
+
+      # @api public
+      def group_and_count(*args, &block)
+        __new__(dataset.__send__(__method__, *args, &block))
+      end
+
+      # @api public
+      def select_append(*args, &block)
+        __new__(dataset.__send__(__method__, *args, &block))
+      end
+
+      # @api public
+      def select_group(*args, &block)
+        __new__(dataset.__send__(__method__, *args, &block))
+      end
```

Comment:
They're being defined separately so they can be documented. But I guess the `__new__(dataset.__send__(__method__, *args, &block))` lines "DRY'ed" a bit.